web/src/layout/main.jsx

```diff
@@ -62,6 +63,12 @@ const NewVersionButton = styled.button`
   cursor: pointer;
 `;
 
+const ToggleWrapper = styled.div`
```

Comment:
Creative idea: instead of calling them wrappers we could call them something like 'HiddenOnMobile' so it's more easily readable

Reply:
Yeah I just remember Material UI had some functionality like this but it seems it is not deprecated

https://mui.com/material-ui/migration/v5-component-changes/#hidden

Reply:
Ah, I see - I guess that could make sense, hope we can simplify some of these "mobile" vs "desktop" things later on :)